CHANGES.md

```diff
@@ -35,9 +35,11 @@ To be released.
      -  Added `ITransaction` interface.
  -  Added `IActionTypeLoader` interface.  [[#2539]]
  -  Added `StaticActionTypeLoader` class.  [[#2539]]
+ -  (Libplanet.Explorer) Added `--schema` option-like subcommand.  [[#2563]]
```

Comment:
I think this should be placed in another section like CLI tools.  🤔

Reply:
I moved its section! ✅